ydb/core/fq/libs/compute/ydb/control_plane/compute_database_control_plane_service.cpp

```diff
@@ -38,6 +38,8 @@ struct TDatabaseClients {
         TActorId ActorId;
         NConfig::TComputeDatabaseConfig Config;
         TActorId DatabasesCacheActorId;
+        TActorId MonitoringClientActorId;
```

Comment:
We don't use this variable. Can we get rid of it?

Reply:
It was with graceful shutdown/cleanup in mind.